### PR TITLE
Create da.json

### PR DESCRIPTION
**Pull request by prathers**
Danish translation

### CODE REVIEW

File: i18n/da.json [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+    "campaigns.schedule": "Planlæg kampagne",
+    "campaigns.scheduled": "Planlagt",
+    "campaigns.send": "Sende",
+    "campaigns.sendLater": "Send senere",
+    "campaigns.sendTest": "Send testmeddelelse",
+    "campaigns.sendTestHelp": "Tryk på Enter efter at have indtastet en adresse for at tilføje flere modtagere. Adresserne skal tilhøre eksisterende abonnenter.",
+    "campaigns.sendToLists": "Lister, der skal sendes til",
+    "campaigns.sent": "Sendt",
+    "campaigns.start": "Start kampagne",
+    "campaigns.started": "\{name}\" startet",
```
Comment (knadh) [inline i18n/da.json:83]: This, and a few instances below, have broken quotes. Should be `"\"{name}\" startet"`

Comment (prathers) [reply i18n/da.json:83]: Let me check

**Review by kevinanielsen — APPROVED** [review]
LGTM

File: i18n/da.json [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+    "templates.errorRendering": "Fejlmeddelelse om fejlgengivelse: {error}",
+    "templates.fieldInvalidName": "Ugyldig længde for navn.",
+    "templates.makeDefault": "Indstil standard",
+    "templates.newTemplate": "Ny skabelon",
+    "templates.placeholderHelp": "Pladsholderen {placeholder} skal vises nøjagtigt én gang i skabelonen.",
+    "templates.preview": "Forhåndsvisning",
+    "templates.rawHTML": "Rå HTML",
+    "templates.subject": "Emne",
+    "users.login": "Log ind",
+    "users.logout": "Log ud",
```
Comment (knadh) [inline i18n/da.json:579]: This trailing comma is breaking the JSON and should be removed. Last issue to fix!

You can validate JSON using a site like https://www.jsonlint.com/

**Pull request merged**